code/_onclick/click.dm

```diff
@@ -144,12 +144,16 @@
 	SEND_SIGNAL(src, COMSIG_MOB_POST_CLICK, A, mods)
 	return
 
+#define ATTACKBY_HINT_NO_AFTERATTACK (1 << 0)
+#define ATTACKBY_HINT_UPDATE_NEXT_MOVE (1 << 1)
```

Comment:
Put that in define files not out there randomly, especially since they're used in multiple places